### PR TITLE
Add pytest-cov doc

### PR DESCRIPTION
**Pull request by grindtildeath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -290,6 +290,12 @@ docker-compose run --rm odoo dropdb testdb
 Pytest uses a plugin (https://github.com/camptocamp/pytest-odoo) that corrects the
 Odoo namespaces (`openerp.addons`/`odoo.addons`) when running the tests.
 
+### pytest-cov
+
+pytest-cov is also included and can be used to generate a coverage report. 
+You can simply add --cov=MODULE_PATH to your pytest to get a text version in the shell, or export it as HTML so you can browse the results.
```
Comment (guewen) [inline README.md]:
```suggestion
You can add --cov=MODULE_PATH to your pytest to get a text version in the shell, or export it as HTML so you can browse the results.
```

File: README.md [hunk 2]
```diff
@@ -290,6 +290,12 @@ docker-compose run --rm odoo dropdb testdb
...
+To export it to HTML, simply add --cov-report=HTML:EXPORT_PATH
```
Comment (guewen) [inline README.md]:
```suggestion
To export it to HTML, add --cov-report=HTML:EXPORT_PATH
```

File: README.md [hunk 3]
```diff
@@ -290,6 +290,12 @@ docker-compose run --rm odoo dropdb testdb
 Pytest uses a plugin (https://github.com/camptocamp/pytest-odoo) that corrects the
 Odoo namespaces (`openerp.addons`/`odoo.addons`) when running the tests.
 
+### pytest-cov
+
+pytest-cov is also included and can be used to generate a coverage report. 
+You can add --cov=MODULE_PATH to your pytest to get a text version in the shell, or export it as HTML so you can browse the results.
```
Comment (simahawk) [inline README.md]: should mention that `MODULE_PATH` is `odoo.addons.$module_name` and that you can use `--cov-report=term-missing` (recommended).

File: README.md [hunk 4]
```diff
@@ -290,6 +290,12 @@ docker-compose run --rm odoo dropdb testdb
...
+To export it to HTML, add --cov-report=HTML:EXPORT_PATH
```
Comment (yvaucher) [inline README.md]: `EXPORT_PATH` is a location into the docker container? Would it be worth mentioning the need to add a volume?

**Pull request merged**